packages/flutter_tools/templates/widget_test/widget_test.dart.tmpl

```diff
@@ -0,0 +1,44 @@
+// This is a basic Flutter widget test.
```

Comment:
We should make sure that running `flutter test` on this test actually passes. Perhaps add it to https://github.com/flutter/flutter/blob/master/dev/bots/test.dart?